### PR TITLE
Version 4.4

### PR DESCRIPTION
**Pull request by rspeer**
Uses the new heuristics, the ability to fix Windows-1250 and ISO-8859-2, and a change to the order of fixers.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+## Version 4.4.0 (February 17, 2017)
+
+Heuristic changes:
+
+- ftfy can now fix mojibake involving the Windows-1250 or ISO-8859-2 encodings.
+
+- The `fix_entities` fixer is now applied after `fix_encoding`. This makes
+  more situations resolvable when both fixes are needed.
```
Comment (alin-luminoso) [inline CHANGELOG.md]: Is it just not likely that something was mojibaked and then HTML-escaped?

File: setup.py [hunk 2]
```diff
@@ -38,21 +38,3 @@
...
-    The next major version of ftfy, version 5.0, will probably only work
-    on Python 3, making it easier to develop and ensure its consistency.
-
-    It's fine if you're happy with Python 2 and ftfy 4. Save yourself a
-    headache later and be sure to pin a version in your dependencies.
-    Instead of just asking for 'ftfy', you should ask for:
-
-        'ftfy >= 4, < 5'
-    """.strip(' '))
-
```
Comment (alin-luminoso) [inline setup.py]: Why this removal?

Comment (rspeer) [inline setup.py]: 5.0 is going to be released at the same time, so if you got version 4.4, you already heeded the warning.

File: CHANGELOG.md (hunk 1 above)
Comment (rspeer) [inline CHANGELOG.md]: The main `fix_text` function iterates until it reaches a fixed point, so this should still work. The problem was that I saw a real case where decoding HTML first created a character that wasn't in the encoding, so the encoding fixer got stuck.

**Pull request merged**